### PR TITLE
Move the config template write to the first actions to be done

### PR DESCRIPTION
**Pull request by mvangoor**
# Description

The postfix config is written before we try to execute postfix commands (like postmap)

## Issues Resolved

With this change setting inet_interfaces to ipv4 works and commands will work on ipv6 disabled environments.

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This file is used to list changes made in each version of the postfix cookbook.
 
 ## Unreleased
 
+- postfix package assumes ipv6 is always available, however this is not always true. In order to tell postmap this is the case the configs need to be written first
```
Comment (ramereth) [inline CHANGELOG.md:7]: Can you please shorten this? It's a bit long and I think what you used as the subject for this PR would work well.

**Review by ramereth — CHANGES_REQUESTED** [review]
We're waiting on a new release of Chef Workstation to resolve the CI failures but LGTM other than what I mentioned.

File: CHANGELOG.md (hunk 1 above)
Comment (mvangoor) [reply CHANGELOG.md:7]: Sure, let me ammend the PR

**Review by ramereth — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [6.0.2](https://github.com/sous-chefs/postfix/releases/tag/6.0.2)

**Pull request merged**